src/pages/p/[token]/complain/index.astro

```diff
@@ -36,27 +21,5 @@ if (proceeding.state !== 'awaitingComplaint') return new Response('You cannot se
 const state = proceeding.complaintState;
 if (!state || !isValidComplaintState(state)) throw new Error('This should never happen.');
 
-const { developerName, privacyPolicyUrl, complaintAuthority } = proceeding;
+return Astro.redirect(`/p/${token}/complain/${state}`);
```

Comment:
I like that we're keeping the alias and it always redirects you to the correct page, but we should still update all existing links here to avoid the unnecessary redirect.

Reply:
Ah. I see now that you already did this in most places and only missed one:

https://github.com/tweaselORG/platform/blob/1bff9480ccbba0a75a6fbe7d08eaba520d40123d/src/components/ProceedingLandingPage/AwaitingComplaint.astro#L39